src/orders/components/OrderTransaction/components/CardTitle/CardTitle.tsx

```diff
@@ -70,88 +90,88 @@ export const OrderTransactionCardTitle = ({
   } = transaction;
 
   const actions = transaction.actions.filter(action => action !== TransactionActionEnum.REFUND);
-  const showActionButtons = showActions && actions.length > 0;
+  const showActionsMenu = showActions && actions.length > 0;
 
-  return (
-    <Box width="100%" display="flex" justifyContent="space-between" alignItems="center">
-      {transaction.externalUrl ? (
-        <ButtonLink
-          as="a"
-          href={transaction.externalUrl}
-          display="flex"
-          gap={2}
-          alignItems="center"
-          size="large"
-          __flex={1}
-        >
-          <ExternalLink size={iconSize.small} strokeWidth={iconStrokeWidth} />
-
-          <TransactionTitle transaction={transaction} index={index} />
-        </ButtonLink>
-      ) : (
-        <Text size={3} fontWeight="bold" __flex={1}>
-          <TransactionTitle transaction={transaction} index={index} />
-        </Text>
-      )}
-
-      <Box display="flex" gap={8} alignItems="center">
-        {cancelPendingAmount.amount > 0 && (
-          <MoneyDisplay
-            label={intl.formatMessage(messages.cancelPending)}
-            money={cancelPendingAmount}
-          />
-        )}
-
-        {canceledAmount.amount > 0 && (
-          <MoneyDisplay label={intl.formatMessage(messages.canceled)} money={canceledAmount} />
-        )}
-
-        {refundPendingAmount.amount > 0 && (
-          <MoneyDisplay
-            label={intl.formatMessage(messages.refundPending)}
-            money={refundPendingAmount}
-          />
-        )}
+  const isDestructiveAction = (action: TransactionActionEnum) =>
+    action === TransactionActionEnum.CANCEL || action === TransactionActionEnum.REFUND;
```

Comment:
issue: This should be declared outside of component, to avoid re-creating function on each render. We can move it to separate file or above this component definition.